pkg/ucp/frontend/controller/planes/listplanes.go

```diff
@@ -54,28 +54,43 @@ func (e *ListPlanes) Run(ctx context.Context, w http.ResponseWriter, req *http.R
 	serviceCtx := v1.ARMRequestContextFromContext(ctx)
 	logger := ucplog.FromContextOrDiscard(ctx)
 
-	query := store.Query{
-		RootScope:    serviceCtx.ResourceID.String(),
-		IsScopeQuery: true,
+	// The plane objects are all stored separately (by plane type). We need to query each type separately.
+	planeTypes := []string{
+		"aws",
+		"azure",
+		"radius",
 	}
-	logger.Info(fmt.Sprintf("Listing planes in scope %s", query.RootScope))
-	result, err := e.StorageClient().Query(ctx, query)
-	if err != nil {
-		return nil, err
+
+	objs := []store.Object{}
+	for _, planeType := range planeTypes {
+		query := store.Query{
+			RootScope:    serviceCtx.ResourceID.String(),
+			ResourceType: planeType,
+			IsScopeQuery: true,
```

Comment:
This is the one place where we did a "multi-type query". eg: the only place besides tests where I had to change the code.